HISTORY.rst

```diff
@@ -8,6 +8,7 @@ Pending release
 
 * New release notes go here
 * Fix django session keys not being fingerprinted.
+* Add settings for assertion mode.
```

Comment:
I think a bit more of an explanation here would be good, this is a major feature and users who are upgrading might be keen to use it immediately